src/chats/components/conversation/messageBubbles/AttachmentView.test.tsx

```diff
@@ -40,6 +40,22 @@ describe('Attachment view', () => {
 		expect(mockedGetURLAttachment).toHaveBeenCalled();
 	});
 
+	test('file with long mimeType visualization', () => {
```

Comment:
Also add a test like this to check all the extensions

`
const attachmentCases = [
	[AttachmentType.GIF, 'image', AttachmentType.GIF],
	[AttachmentType.PNG, 'image', AttachmentType.PNG],
	[AttachmentType.JPEG, 'image', AttachmentType.JPEG],
	[AttachmentType.WEBP, 'image', AttachmentType.WEBP],
	[AttachmentType.PDF, 'application', AttachmentType.PDF],
	[
		AttachmentType.DOCX,
		'application',
		'vnd.openxmlformats-officedocument.wordprocessingml.document'
	],
	[
		AttachmentType.PPTX,
		'application',
		'vnd.openxmlformats-officedocument.presentationml.presentation'
	],
	[AttachmentType.XLSX, 'application', 'vnd.openxmlformats-officedocument.spreadsheetml.sheet'],
	[AttachmentType.ODP, 'application', 'vnd.oasis.opendocument.presentation'],
	[AttachmentType.ODS, 'application', 'vnd.oasis.opendocument.spreadsheet'],
	[AttachmentType.ODT, 'application', 'vnd.oasis.opendocument.text'],
	[AttachmentType.MPKG, 'application', 'vnd.apple.installer+xml'],
	[AttachmentType.PPT, 'application', 'vnd.ms-powerpoint'],
	[AttachmentType.XLS, 'application', 'vnd.ms-excel']
];


test.each(attachmentCases)('display %s extension', (ext, type, mimetype) => {
		const genericAttachment: AttachmentMessageType = {
			id: 'genericAttachmentId',
			name: `generic.${ext}`,
			mimeType: `${type}/${mimetype}`,
			size: 21412
		};
		setup(<AttachmentView attachment={genericAttachment} from={'from'} />);
		const genericIcon = screen.getByTestId('icon: FileTextOutline');
		expect(genericIcon).toBeVisible();
		const fileName = screen.getByText(genericAttachment.name);
		expect(fileName).toBeVisible();
		const match = new RegExp(`${ext}`, 'i');

		const extension = screen.getByText(match);
		expect(extension).toBeInTheDocument();
	});`

Reply:
`const attachmentCases = [
	[AttachmentType.GIF, 'image', AttachmentType.GIF],
	[AttachmentType.PNG, 'image', AttachmentType.PNG],
	[AttachmentType.JPEG, 'image', AttachmentType.JPEG],
	[AttachmentType.WEBP, 'image', AttachmentType.WEBP],
	[AttachmentType.PDF, 'application', AttachmentType.PDF],
	[
		AttachmentType.DOCX,
		'application',
		'vnd.openxmlformats-officedocument.wordprocessingml.document'
	],
	[
		AttachmentType.PPTX,
		'application',
		'vnd.openxmlformats-officedocument.presentationml.presentation'
	],
	[AttachmentType.XLSX, 'application', 'vnd.openxmlformats-officedocument.spreadsheetml.sheet'],
	[AttachmentType.ODP, 'application', 'vnd.oasis.opendocument.presentation'],
	[AttachmentType.ODS, 'application', 'vnd.oasis.opendocument.spreadsheet'],
	[AttachmentType.ODT, 'application', 'vnd.oasis.opendocument.text'],
	[AttachmentType.MPKG, 'application', 'vnd.apple.installer+xml'],
	[AttachmentType.PPT, 'application', 'vnd.ms-powerpoint'],
	[AttachmentType.XLS, 'application', 'vnd.ms-excel']
];`

Reply:
`test.each(attachmentCases)('display %s extension', (ext, type, mimetype) => {
		const genericAttachment: AttachmentMessageType = {
			id: 'genericAttachmentId',
			name: `generic.${ext}`,
			mimeType: `${type}/${mimetype}`,
			size: 21412
		};
		setup(<AttachmentView attachment={genericAttachment} from={'from'} />);
		const genericIcon = screen.getByTestId('icon: FileTextOutline');
		expect(genericIcon).toBeVisible();
		const fileName = screen.getByText(genericAttachment.name);
		expect(fileName).toBeVisible();
		const match = new RegExp(`${ext}`, 'i');

		const extension = screen.getByText(match);
		expect(extension).toBeInTheDocument();
	});`